app/src/main/java/nl/tudelft/trustchain/app/service/TrustChainService.kt

```diff
@@ -7,17 +7,17 @@ import androidx.core.app.NotificationCompat
 import androidx.core.app.TaskStackBuilder
 import nl.tudelft.ipv8.android.service.IPv8Service
 import nl.tudelft.trustchain.app.R
-import nl.tudelft.trustchain.explorer.ui.TrustChainExplorerActivity
+import nl.tudelft.trustchain.app.ui.dashboard.DashboardActivity
 
```

Comment:
```suggestion

@RequiresApi(Build.VERSION_CODES.M)
```

Reply:
Because we now switched to an AppCompatActivity, we need at least API 23.

Reply:
Done through a new commit, thanks!